WORKSPACE

```diff
@@ -51,3 +51,12 @@ http_archive(
     strip_prefix = "googletest-release-1.10.0",
     urls = ["https://github.com/google/googletest/archive/release-1.10.0.tar.gz"],
 )
+
+http_archive(
+    # TODO(@mathetake): use non-prebuild library, or download separate archives for each architecture
```

Comment:
I believe this is only used in this WORKSPACE for testing and it could be left

Reply:
We need to add and build an equivalent target in Envoy, so this needs to be sorted out.

Reply:
OK, will do.

Reply:
I'm almost there with a few tricky errors. working repo: https://github.com/mathetake/bazel-wasmtime-c-api

Reply:
completed